packages/dialtone-icons/transformSVGtoVue.cjs

```diff
@@ -2,42 +2,88 @@ const fs = require('node:fs');
+
+        if (!/\${uniqueID}/g.test(result)) {
+          // Remove created function if not needed
+          result = result.replace(/\n\s+this.uniqueID.*;/, '');
+          result = result.replace(/import \{ getUniqueString \} from '\.\.\/utils';\n+/, '');
+        }
+
+        fs.writeFileSync(`./src/icons/${fileName.replace('.svg', '.vue')}`, result, 'utf8');
+    })
+  })
 
-      let result = template.replace('__SVG_CONTENT__', svgContent);
-      result = result.replaceAll('__ICON_NAME__', iconName);
+  /**
+   * ILLUSTRATIONS BUILD
+   */
 
-      // result = result.replace('<svg', '<svg :aria-label="ariaLabel" :class="[iconSizeClass]"')
-      result = result.replace('<svg', '<svg :aria-label="ariaLabel" :class="iconSizeClass"')
+  if (!fs.existsSync('./src/illustrations')) fs.mkdirSync('./src/illustrations');
 
-      result = result.replace('aria-hidden="true"', ':aria-hidden="ariaHidden"')
+  fs.readdir('./dist/svg/illustrations', function (err, fileNames) {
+    if (err) {
+      console.log(err);
+      return -1;
+    }
 
-      // Create unique IDs
-      result = result.replace(/(clip-path|fill)="url\(#([^)]+)\)"/g, ':$1="`url(#${uniqueID}$2)`"')
-      result = result.replace(/(clipPath|linearGradient|radialGradient) id="([^"]+)"/g, '$1 :id="`${uniqueID}$2`"')
+    fileNames.forEach(function (fileName) {
+        const svgContent = fs.readFileSync(`./dist/svg/illustrations/${fileName}`, 'utf8');
+        const template = fs.readFileSync('./src/IllustrationTemplate.vue', 'utf8');
+        const illustrationName = `dt-illustration-${fileName.replace('.svg', '')}`.toLowerCase()
+        .split('-')
+        .map(word => word.charAt(0).toUpperCase() + word.slice(1))
+        .join('');
 
-      if (!/\${uniqueID}/g.test(result)) {
-        // Remove created function if not needed
-        result = result.replace(/\n\s+this.uniqueID.*;/, '');
-        result = result.replace(/import \{ getUniqueString \} from '\.\.\/utils';\n+/, '');
-      }
+        let result = template.replace('__SVG_CONTENT__', svgContent);
+        result = result.replaceAll('__ILLUSTRATION_NAME__', illustrationName);
 
-      fs.writeFileSync(`./src/icons/${fileName.replace('.svg', '.vue')}`, result, 'utf8');
+        result = result.replace('<svg', '<svg :aria-label="ariaLabel"')
+
+        result = result.replace('aria-hidden="true"', ':aria-hidden="ariaHidden"')
+
+        // // Create unique IDs
+        // result = result.replace(/(clip-path|fill)="url\(#([^)]+)\)"/g, ':$1="`url(#${uniqueID}$2)`"')
+        // result = result.replace(/(clipPath|linearGradient|radialGradient) id="([^"]+)"/g, '$1 :id="`${uniqueID}$2`"')
+        //
+        // if (!/\${uniqueID}/g.test(result)) {
+        //   // Remove created function if not needed
+        //   result = result.replace(/\n\s+this.uniqueID.*;/, '');
+        //   result = result.replace(/import \{ getUniqueString \} from '\.\.\/utils';\n+/, '');
+        // }
```

Comment:
Removed from illustrations components since it is affecting the svg.
The svg, does not work anymore after doing this. Probably something with the regex.

Reply:
This doesn't modify the SVG, only the Vue component output.

It's only reading the files, never writing the SVG's back, only the vue components.

Reply:
👍🏼 , solved, was confused cos it grabs the already minifed one that is on `/dist/` folder.
We removed the minifed process from the `.pipe`.